cpp/lib/ome/test/main.cpp

```diff
@@ -45,3 +45,11 @@ main (int   argc,
   testing::InitGoogleTest(&argc, argv);
   return RUN_ALL_TESTS();
 }
+
+bool
+verbose ()
+{
+  static bool v(getenv("BIOFORMATS_TEST_VERBOSE")
+                && std::string(getenv("BIOFORMATS_TEST_VERBOSE")) == "true");
```

Comment:
This is only place this is used (at present).  It could be made more general if needed (anything that would parse as true), but at the moment it's purely for use by anyone running the tests by hand when debugging the test framework.